app/assets/stylesheets/hyrax/dashboard.scss

```diff
@@ -74,12 +74,6 @@ body.dashboard {
    * replace the "navbar-static-top" class with "navbar-fixed-top" for the Admin UI only
    */
   #masthead {
-    border: 0;
-    left: 0;
-    position: fixed;
-    right: 0;
-    top: 0;
-
```

Comment:
I fought with removing and adding this a few times.  Glad to see you got the `fixed-top` implementation going.